test/TestUtilities/GraphicsUtil.hpp

```diff
@@ -23,6 +25,7 @@ namespace sf
     template <typename T>
     std::ostream& operator <<(std::ostream& os, const sf::Rect<T>& rect)
     {
+        os << std::fixed << std::setprecision(std::numeric_limits<T>::max_digits10);
```

Comment:
I am late, but precision in not a flag. It can be queried and reset with the [precision](https://en.cppreference.com/w/cpp/io/ios_base/precision) method.

Reply:
How would you recommend we temporarily change precision just for this one operator?

Reply:
```c++
const auto precisionBefore = os.precision(std::numeric_limits<T>::max_digits10);

// output stuff...

os.precision(precisionBefore);
```